docusaurus/docs/Android/04-compose/04-message-components/01-messages-screen.mdx

```diff
@@ -53,42 +54,40 @@ fun MessagesScreen(
 Here's an example of setting up custom behavior for the user tapping on the back button and the channel info:
 
 ```kotlin
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        val channelId = intent.getStringExtra(KEY_CHANNEL_ID) ?: return
-
-        setContent {
-            ChatTheme {
-                MessagesScreen(
-                    ..., // State
-                    onBackPressed = { finish() }, // Navigation handler
-                    onHeaderActionClick = { channel ->
-                        // Show conversation info
-                    },
-                )
-            }
-        }
-    }
+MessagesScreen(
+    ..., // State
+    onBackPressed = { finish() }, // Navigation handler
+    onHeaderActionClick = { channel ->
+        // Show conversation info
+    },
+)
 ```
 
 ## Customization
 
-Given that the `MessagesScreen` is an screen level solution, it offers limited customization. Currently, the only option offered in the signature is the following:
+Given that the `MessagesScreen` is a screen level solution, it offers limited customization. Currently, the only options offered in the signature are the following:
 
 ```kotlin
 @Composable
 fun MessagesScreen(
+    messageLimit: Int = 30,
     showHeader: Boolean = true,
     ... // action handlers and state
```

Comment:
What about `enforceUniqueReactions`? What do you think, should we mention them?

Reply:
That is a very valid suggestion and an important part that I had probably overlooked by accident.

I will fix that.